packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -376,6 +376,7 @@ export default class ScriptTransformer {
       const sourceMapContent =
         typeof map === 'string' ? map : JSON.stringify(map);
       writeCacheFile(sourceMapPath, sourceMapContent);
+      code += '\n//# sourceMappingURL=' + sourceMapPath;
```

Comment:
my one concern is - what if there's already a mapping url? or is that extracted by the transforms?

Reply:
It won't be a problem. Whichever sourcemap (inline or external) is used, they are generated by the same tool. In the case of [`convert-source-map`](https://github.com/thlorenz/convert-source-map), the first `sourceMappingURL` comment is used. This patch ensures transformed code without a `sourceMappingURL` has one by the time `Runtime._execModule` is reached.

Reply:
cool, thanks!